actors/miner/src/lib.rs

```diff
@@ -4197,6 +4197,7 @@ fn request_terminate_deals(
         ));
         // If running in a system / cron context intentionally swallow this error to prevent
         // frozen market cron corruption from also freezing this miner cron.
+        // TODO: Should we instead check that immediate caller is POWER?
```

Comment:
Don't think it really matters, will drop if we decide no.

Reply:
Note this code has two entry paths, and one of them is a user call. But then either check is fine, I don't think we need a TODO. You could note checking the caller as an alternative.

Reply:
I did it this way because Id want us to maintain the same behavior for any implicit message as those are the ones where failures get really problematic and implicit messages conventionally come from the system actor.

That said I don't expect a lot of change here for a while and am totally happy with it looking for cron as latest caller too.

Reply:
Ack, that makes sense -- I'll leave it at SYSTEM and drop the TODO, I think I agree with your reasoning.